runtime/tcpsrv.c

```diff
@@ -1000,6 +1000,7 @@ BEGINobjConstruct(tcpsrv) /* be sure to specify the object type also in END macr
 	pThis->bUseFlowControl = 1;
 	pThis->pszDrvrName = NULL;
 	pThis->bPreserveCase = 1; /* preserve case in fromhost; default to true. */
+	pThis->DrvrTlsVerifyDepth = 25;
```

Comment:
This looks inconsistent to the other default. I would suggest to leave the default at 0, as this avoids the API call and so the library default is used (whatever it is). I think this is the least confusing option for users.